tests/llms/test_llms_text-generation_vllm_langchain_openvino_on_intel_arc.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -x
+
+WORKPATH="$( cd "$( dirname "$0" )" && pwd )"
+
+# Define variables
+port=5033
+RENDER_GROUP_ID=110
+HF_MODEL_FIR=$HOME/hf_model
+DOCKER_IMAGE="vllm-openvino:comps"
+CONTAINER_NAME="test-comps-vllm-openvino-container"
+
+function build_container() {
+    cd $WORKPATH
+    git clone https://github.com/vllm-project/vllm.git vllm-openvino
+    cd ./vllm-openvino
+
+    git reset --hard 067e77f9a87c3466fce41c8fe8710fddc69ec26c # resolve circular import issue
+
+    # Add ARC drive to dockerfile
```

Comment:
Should we add a dockerfile into https://github.com/opea-project/GenAIComps/tree/9473aa44befbe04ce3799000a0f83c513654570d/comps/llms/text-generation/vllm/langchain?